src/pages/app/posting_project_user/styles/ProfilePage.css

```diff
@@ -10,18 +10,18 @@
 
 .profilePage {
   color: var(--title);
-  background-color: var(--secondary);
+  background-color: var(--global-page-background-color);
   border-radius: none;
   vertical-align: middle;
-  font-family: "Inter";
 }
 
 /*Component*/
 .profile--title {
   color: var(--title);
-  font-size: 150%;
+  font-size: 20px;
```

Comment:
**Global style changes needed**

Reply:
I changed the --title. not sure about the font-size though... will have to check it again